### PR TITLE
Information/CenterOfMass: Fix typos and whitespace

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: Information/CenterOfMass.FCMacro [hunk 1]
```diff
@@ -596,13 +601,13 @@ class CenterofmassWidget(QtGui.QWidget):
         tu = FreeCAD.Units.parseQuantity
         if defDensity<self.defaultDensitySpin.maximum() and defDensity>self.defaultDensitySpin.minimum():
             self.material_database['default'] = float(tu('{}'.format(defDensity)+self.comboUnit.currentText()) / tu('kg/m^3'))
-    
+
     def setalltodefaultdensity_stateChanged(self, state):
         for sol in range(self.solid_count):
             self.materials[sol].combo.setCurrentIndex(0)
-            self.materials[sol].combo.setCurrentText('default');                    
+            self.materials[sol].combo.setCurrentText('default');
```
Comment (galou) [inline Information/CenterOfMass.FCMacro]: Optional: extra semi-colon.

File: Information/CenterOfMass.FCMacro [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+# Usage:
+# 1. select one or multiple solids
+# 2. launch the macro.
+# 3. You'll have a window listing the solids. You can put the density of your
+#    material or choose from a large list of predefined materials. (If you know
+#    your densities in another unit system, I used FreeCAD convert system to
+#    propose different units.)
 #
+# Options:
+$ * You can also display where is the center of mass.
```
Comment (galou) [inline Information/CenterOfMass.FCMacro]: I see a dollar sign in the diff. What is it?

Comment (luzpaz) [reply Information/CenterOfMass.FCMacro]: yikes, probably something i introduced

File: Information/CenterOfMass.FCMacro (hunk 1 above)
Comment (luzpaz) [reply Information/CenterOfMass.FCMacro]: @galou like:   
`self.materials[sol].combo.setCurrentText('default');;`  
Or are you referring to line 602 ?

Comment (galou) [reply Information/CenterOfMass.FCMacro]: I thought `self.materials[sol].combo.setCurrentText('default')`, i.e. without semi-colon.

Comment (luzpaz) [reply Information/CenterOfMass.FCMacro]: Done in 04011ae

**Pull request merged**